api/order/orderRoute.js

```diff
@@ -1 +1,96 @@
-//order endpoints
+import express from 'express';
+import Order from './Order.js';
+import User from '../user/User.js';
+import Ticket from '../ticket/Ticket.js';
+
+const router = express.Router();
+
+router
+  .route('/')
+  .get(async (req, res) => {
+    const orders = await Order.find({});
+    res.status(200).json(orders);
+  })
+  .post(async (req, res) => {
```

Comment:
This should be available for authenticated user? or any user who paid? I guess we shuold evaluate